server/src/routes/trpc/students.ts

```diff
@@ -0,0 +1,40 @@
+import { z } from "zod";
+import { t } from "./context";
+
+// todo: define .output() schemas for all procedures
```

Comment:
nit: I recommend using this convention for TODOs: https://google.github.io/styleguide/pyguide.html#312-todo-comments